tgui/packages/tgui/interfaces/SlotMachine.js

```diff
@@ -0,0 +1,58 @@
+import { useBackend } from "../backend";
+import { Button, LabeledList, Box, AnimatedNumber } from "../components";
+import { Window } from "../layouts";
+
+export const SlotMachine = (props, context) => {
+  const { act, data } = useBackend(context);
+  if (data.money === null) {
+    return (
+      <Window>
+        <Window.Content>
+          <Box m={1}>
+            <Box>
+              Could not scan your card or could not find account!
+            </Box>
+            <Box>
+              Please wear or hold your ID and try again.
+            </Box>
+          </Box>
+        </Window.Content>
+      </Window>
+    );
+  } else {
+    let playerText;
+    if (data.plays === 1) {
+      playerText = data.plays + " player has tried their luck today!";
+    } else {
+      playerText = data.plays + " players have tried their luck today!";
+    }
```

Comment:
You can freely create variables inside the UI as you would in JavaScript, allowing for custom messages based off of a value. Here I have used that to make it grammatically correct when only one person has spun the slot machine.